.travis.yml

```diff
@@ -0,0 +1,95 @@
+language: cpp
+cache: ccache
+os:
+  - linux
+  - osx
+dist: xenial
+osx_image: xcode10.1
+compiler:
+  - clang
+  - gcc
+addons:
+  apt:
+    packages:
+      - cmake
+      - doxygen
+      - libpng-dev
+      - libxml2-utils
+      - llvm
+      - clang
+      - docutils-common
+      - libomp-dev
+      - lcov
+  homebrew:
+    packages:
+      - doxygen
+      - glib
+      - libpng
+      - libxml2
+      - ccache
+      - libomp
+notifications:
+  webhooks:
+    - https://coveralls.io/webhook
+env:
+  global:
+    - COVERALLS_PARALLEL=true
+  matrix:
+    - build_type=release
```

Comment:
Should be `Release`.

Reply:
Cmake doesn't care for caps for build type